cmd/routing-api/testrunner/db.go

```diff
@@ -240,5 +240,6 @@ func (a *sqliteAllocator) minConfig() *config.SqlDB {
 	return &config.SqlDB{
 		Type:   "sqlite3",
 		Schema: a.schemaName,
+		StorePath: ".",
```

Comment:
This seems harder to upstream. Why not just let the scheme name be the full path? Schema is the database isolation, in SQLite that isolation is a file on disk. We used the schema name in our manual standup up to set the full path to somewhere writeable.